chainer/links/connection/mlp_convolution_2d.py

```diff
@@ -44,8 +44,7 @@ class MLPConvolution2D(link.ChainList):
     """
 
     def __init__(self, in_channels, out_channels, ksize, stride=1, pad=0,
-                 activation=relu.relu, wscale=1,
-                 conv_init=None, bias_init=None):
+                 activation=relu.relu, conv_init=None, bias_init=None):
```

Comment:
I am thinking how we should do. The problem in my mind is the following case:
`MLPConvolution2D(10, 3, 3, 1, 0, relu.relu, 1)`
It works without errors, although its semantic change. The problem is we cannot tell which API users expect.